### DISCUSSION TITLE
有关接口返回错误代码的问题

### DISCUSSION THREAD

**Discussion by DaliBerr** [Nemo2011/bilibili-api#88] [Q&A] [answered]
尝试克隆他人关注列表时，关注几个之后报错 ： “接口返回错误代码：22015，信息：您的账号异常，请稍后再试。”
请问是什么问题

**Comment by Nemo2011** [comment, marked as answer]
可能是调用过快，试一下调慢一点

**Reply by DaliBerr** [reply]
起初调的是5秒，后来改成60都并不行，也尝试换ip调用也不行，估计是号被限制了，sad
